incubator/traefik/README.md

```diff
@@ -0,0 +1,86 @@
+# Traefik
+
+[Traefik](http://traefik.io/) is a modern HTTP reverse proxy and load balancer made to deploy
+microservices with ease.
+
+## Introduction
+
+This chart bootstraps Traefik as a Kubernetes ingress controller with optional support for SSL and
+Let's Encrypt.
+
+__N.B.__: Operators will typically wish to install this component into the `kube-system` namespace
+where that namespace's default service account will ensure adequate privileges to watch `Ingress`
+resources _cluster-wide_.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+- You are deploying the chart to a cluster with a cloud provider capable of provisioning an
+external load balancer (e.g. AWS or GKE)
+- You control DNS for the domain(s) you intend to route through Traefik
+
+### Suggested:
+
+- PV provisioner support in the underlying infrastructure
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install incubator/traefik --name my-release --namespace kube-system
+```
+
+After installing the chart, create a DNS CNAME record for applicable domains to direct inbound
+traffic the load balancer. You can you the command below to find the load balancer's DNS name:
```

Comment:
a couple minor typos here; perhaps the following were intended: `... direct inbound traffic to the load balancer` and `You can use the command ...`